packages/fxa-settings/src/components/ConnectedServices/index.tsx

```diff
@@ -69,11 +117,18 @@ export const ConnectedServices = () => {
           />
         </div>
 
-        <Service />
-        <Service />
-        <Service />
-        <Service />
-        <Service />
+        {!!sortedAndUniqueClients.length &&
+          sortedAndUniqueClients.map((client, i) => (
+            <Service
+              {...{
+                key: i,
```

Comment:
Is there another `key` we can use like `clientId`? We shouldn't use the index of an array.

Reply:
Just switched to clientId